tools/driver/swift_format_main.cpp

```diff
@@ -158,11 +158,6 @@ class SwiftFormatInvocation {
       InputFilenames.push_back(A->getValue());
     }
 
-    if (InputFilenames.empty()) {
-      Diags.diagnose(SourceLoc(), diag::error_mode_requires_an_input_file);
```

Comment:
Ah. Most Swift tools use an explicit `-` for this. Is accepting input on stdin better?